controllers/activemqartemis_controller.go

```diff
@@ -163,15 +171,15 @@ func (r *ActiveMQArtemisReconciler) Reconcile(ctx context.Context, request ctrl.
 		}
 	}
 
-	ProcessStatus(customResource, r.Client, request.NamespacedName, *namer, err)
+	common.ProcessStatus(customResource, r.Client, request.NamespacedName, *namer, err)
```

Comment:
ProcessStatus seems only used in controllers/activemqartemis_controller.go, why was it moved?

Reply:
this is also to move those utility func to common to reuse not live in any controllers.